src/enclave/httpendpoint.h

```diff
@@ -132,15 +132,15 @@ namespace enclave
     }
 
     void handle_message(
-      http_method method,
+      http_method verb,
```

Comment:
This (the HTTP verb, e.g. `POST` or `GET`) was ambiguous with CCF's method parsed from `path` a few lines below.